clusterloader2/cmd/clusterloader.go

```diff
@@ -66,8 +66,6 @@ func initClusterFlags() {
 	flags.StringEnvVar(&clusterLoaderConfig.ClusterConfig.EtcdCertificatePath, "etcd-certificate", "ETCD_CERTIFICATE", "/etc/srv/kubernetes/pki/etcd-apiserver-server.crt", "Path to the etcd certificate on the master machine")
 	flags.StringEnvVar(&clusterLoaderConfig.ClusterConfig.EtcdKeyPath, "etcd-key", "ETCD_KEY", "/etc/srv/kubernetes/pki/etcd-apiserver-server.key", "Path to the etcd key on the master machine")
 	flags.IntEnvVar(&clusterLoaderConfig.ClusterConfig.EtcdInsecurePort, "etcd-insecure-port", "ETCD_INSECURE_PORT", 2382, "Inscure http port")
-	flags.BoolEnvVar(&clusterLoaderConfig.ClusterConfig.DeleteStaleNamespaces, "delete-stale-namespaces", "DELETE_STALE_NAMESPACES", false, "Whether to delete all stale namespaces before the test execution.")
```

Comment:
I'm probably fine with deleting those. But it would be great if we could provide a defaulting in a way that would accept the CL2_DELETE_STALE_NAMESPACES env var as overwrite for it.
Didin't think about it much - @mm4tt - do you think it would be possible?

Reply:
To maintain back compatibility,  it might be good to keep it for the time being. I'll undo the deletion.

Reply:
To maintain backward compatibility, the current command line options and environment variables "delete-stale-namespaces" and "delete-automanaged-namespaces" are kept with deprecated warnings.  automananagedNamespaces can still be used to specify the number of namespaces in testconfig file. If a parameter is not specified, a default value is used. If a parameter is specified in both new and old ways, the value using the new way is used.